service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/StatistikkServiceImpl.kt

```diff
@@ -3,23 +3,30 @@ package no.nav.su.se.bakover.service.statistikk
 import no.nav.su.se.bakover.client.kafka.KafkaPublisher
 import no.nav.su.se.bakover.common.objectMapper
 import no.nav.su.se.bakover.common.zoneIdOslo
+import no.nav.su.se.bakover.database.sak.SakRepo
+import no.nav.su.se.bakover.database.vedtak.VedtakRepo
+import no.nav.su.se.bakover.domain.vedtak.Vedtak
 import no.nav.su.se.bakover.service.person.PersonService
 import org.slf4j.LoggerFactory
 import java.time.Clock
 
 internal class StatistikkServiceImpl(
     private val publisher: KafkaPublisher,
     private val personService: PersonService,
+    private val sakRepo: SakRepo,
+    private val vedtakRepo: VedtakRepo,
     private val clock: Clock,
 ) : StatistikkService, EventObserver {
     private val log = LoggerFactory.getLogger(this::class.java)
     private val schemaValidator = StatistikkSchemaValidator
 
+    // TODO: Kalles bare fra handle, burde være private?
```

Comment:
Enig.

Reply:
Tror grunnen att den er public er for att det blir enklere å teste.